components/ingest-service/migration/migration.go

```diff
@@ -15,8 +14,9 @@ import (
 var (
 	// This variable will allow us to add a test so that if the mappings.NodeState.Index
 	// changes we might need to add some migration bits over here
-	a2CurrentNodeStateIndex = "node-state-6" // nolint: varcheck,deadcode
+	a2CurrentNodeStateIndex = "node-state-7" // nolint: varcheck,deadcode
```

Comment:
All the changes on the file add the reindex of the node-state index to create the new "lower" field.